### PR TITLE
Obi 586 - Accessibility on Registration/Login/ReleaseNote

### PR DESCRIPTION
**Pull request by niklasb99**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/components/agencySelection/agencyInfo.styles.scss [hunk 1]
```diff
@@ -11,6 +11,16 @@ $breakpointFlyoutPosition: 1140px;
 			width: $infoIconSize;
 			height: $infoIconSize;
 			cursor: pointer;
+
+			&:focus {
+				outline: 2px solid #199fff !important;
```
Comment (alxbenz) [inline src/components/agencySelection/agencyInfo.styles.scss]: In the other files you use this variable: $focus-outline

File: src/components/agencySelection/agencyLanguages.styles.scss [hunk 2]
```diff
@@ -36,5 +36,15 @@ $input-size: 24px;
 		color: var(--skin-color-primary-contrast-safe, $primary);
 		text-decoration: underline;
 		cursor: pointer;
+
+		&:focus {
+			outline: 2px solid #199fff !important;
```
Comment (alxbenz) [inline src/components/agencySelection/agencyLanguages.styles.scss]: In the other files you use this variable: $focus-outline

File: src/components/agencySelection/AgencyLanguages.tsx [hunk 3]
```diff
@@ -60,6 +60,12 @@ export const AgencyLanguages: React.FC<AgencyLanguagesProps> = ({
 							onClick={() => {
 								setIsAllShown(true);
 							}}
+							tabIndex={0}
+							onKeyDown={(event) => {
```
Comment (alxbenz) [inline src/components/agencySelection/AgencyLanguages.tsx:64]: can we use onKeyPress here?

File: src/components/stageLayout/StageLayout.styles.scss [hunk 4]
```diff
@@ -77,6 +77,20 @@
 		@include breakpoint($fromXLarge) {
 			left: $legal-information-links-left-large;
 		}
+
+		.button-as-link {
+			background: none;
+			border: none;
+
+			&:focus {
+				outline: 2px solid #199fff;
```
Comment (alxbenz) [inline src/components/stageLayout/StageLayout.styles.scss]: $focus-outline

**Review by alxbenz — CHANGES_REQUESTED** [review]

File: src/components/agencySelection/AgencyLanguages.tsx (hunk 3 above)
Comment (niklasb99) [reply src/components/agencySelection/AgencyLanguages.tsx:64]: I don't know exactly why, but on the span element onKeyPress doesn't work

Comment (alxbenz) [reply src/components/agencySelection/AgencyLanguages.tsx:64]: ok... weird

**Review by alxbenz — APPROVED** [review]

**Pull request merged**